tests/snmpsim/ceraos.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.2281.1.20.2.2
```

Comment:
Hi @barryodonovan It probably will. Do you have any other code that may prevent this issue from happening and will detect any CeraOS device no matter what? I could only test with IP-20C as I don't have access to other CeraOS devices unfortunately :(